monkey/infection_monkey/post_breach/pba.py

```diff
@@ -23,8 +24,9 @@ def __init__(self, name="unknown", linux_cmd="", windows_cmd=""):
         """
         self.command = PBA.choose_command(linux_cmd, windows_cmd)
         self.name = name
+        self.pba_data = []
 
-    def run(self):
+    def run(self) -> Iterable[PostBreachData]:
```

Comment:
We also need to make `command` an iterable then, and ideally, each command should be mapped to a result. This would then require a major refactoring in the way telemetry is processed.

Reply:
IDK if it would be that major, only one PBA runs more than one command. Let's look at it once we refactor telemetries